README.md

```diff
@@ -51,18 +188,19 @@ appTags := application.New("OrderingApp", "inventory")
 
 ### 2. Set Up a Wavefront Sender
 
-A "Wavefront sender" is an object that implements the low-level interface for sending data to Wavefront. You can choose to send data using either the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
+A `Wavefront sender` object implements the low-level interface for sending data to Wavefront. You can choose to send data using the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
```

Comment:
Is the object name Wavefront sender (with a space and lower-case s) or WavefrontSender ? Or something different? 

In this case, the best solution might be to say... "A Wavefront sender object (X or Y) " and X and Y are the names of the 2 objects)